search_events_app/tests/services/api/test_api_service.py

```diff
@@ -98,10 +97,10 @@ def setUp(self):
         'format_body'
     )
     @patch(
-        'search_events_app.services.api_service.requests.post'
+        'search_events_app.services.api.api_service.requests.post'
```

Comment:
I think is not needed this newline

Reply:
I don't understand what you mean. We put the files related to the api in a folder called 'api'